pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[build-system]
+requires = ["setuptools>=67.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "sinol-make"
+version = "0.0.1"
+authors = [
+  { name="Mateusz Masiarz", email="m.masiarz@fri.edu.pl" },
+  { name="Tomasz Nowak", email="tomek.nowak.gm.16@gmail.com" }
```

Comment:
I do not want to take credits for your code. This project will be your property and it is more beneficial for you to be the only author (to write about this in the CV, etc). If there is a separate field for maintainers or something like that, it's better to put me there.

Reply:
3119b831ce78324e3399fd78ccc91b64245bb9a7